test/e2e/apps/cloneset.go

```diff
@@ -1071,6 +1072,10 @@ var _ = SIGDescribe("CloneSet", func() {
 			testUpdateVolumeClaimTemplates(tester, randStr, c)
 		})
 
+		ginkgo.It(`change resource and qos -> succeed to recreate`, func() {
```

Comment:
shall we move the case to the suite of InplaceVPA

Reply:
I believe we should design this test case to verify the status of the InPlacePodVerticalScaling feature gate. 

By integrating this test into the InplaceVPA test suite, we ensure that it will only execute when the InPlacePodVerticalScaling feature gate is enabled.